src/CONST.js

```diff
@@ -352,7 +352,6 @@ const CONST = {
         SWITCH_REPORT: 'switch_report',
         SIDEBAR_LOADED: 'sidebar_loaded',
         PERSONAL_DETAILS_FORMATTED: 'personal_details_formatted',
```

Comment:
Should you remove this constant too since it's no longer used?

Reply:
Ah yes, sorry I forgot

Reply:
Done